example-apps/middleware/index.js

```diff
@@ -15,15 +15,17 @@ const addSortingParams = (request /*, z */) => {
 };
 
 // HTTP after middleware that checks for errors in the response.
-const checkForErrors = (response, z) => {
-  // If we get a bad status code, throw an error. This will halt the zap.
-  if (response.status >= 300) {
-    throw new z.errors.HaltedError(
-      `Unexpected status code ${response.status} from ${response.request.url}`
-    );
+const handleErrors = (response, z) => {
+  // Prevent `throwForStatus` from throwing for a certain status.
+  if (response.status === 456) {
+    response.skipThrowForStatus = true;
+  }
```

Comment:
Updated middleware example app to demonstrate new flag, identical to documentation middleware snippet.